keras/layers/convolutional.py

```diff
@@ -1412,9 +1412,15 @@ class ZeroPadding1D(Layer):
     '''Zero-padding layer for 1D input (e.g. temporal sequence).
 
     # Arguments
-        padding: int
+        padding: int or tuple of int (length 2) or dictionary
+            For symmetric padding: int
             How many zeros to add at the beginning and end of
             the padding dimension (axis 1).
+            For asymmetric padding: tuple of int (length 2)
+            How many zeros to add at the beginning and at the end of
+            the padding dimension (left_pad, right_pad) or
+            {'left_pad': left_pad, 'right_pad': right_pad}.
+            If any key is missing, default value of 1 will be used for the missing key.
```

Comment:
Change to 0. Also please format code elements in docstring with "`" (tuples. dicts).